src/ui-services/htmlService.js

```diff
@@ -102,7 +102,7 @@ angular.module("openshiftCommonUI")
         // Replace any URLs with links.
         return text.replace(/https?:\/\/[A-Za-z0-9._%+-]+\S*[^\s.;,(){}<>"\u201d\u2019]/gm, function(str) {
           if (target) {
-            return "<a href=\"" + str + "\" target=\"" + target + "\">" + str + "</a>";
+            return "<a href=\"" + str + "\" target=\"" + target + "\">" + str + " " + "<i class=\"fa fa-external-link\" aria-hidden=\"true\"></i></a>";
```

Comment:
nit: You have an extra `+ " " +` that you don't need.

```js
return "<a href=\"" + str + "\" target=\"" + target + "\">" + str + " <i class=\"fa fa-external-link\" aria-hidden=\"true\"></i></a>";
```

Reply:
My testing indicates we do need the extra space or the icon is butted up against the text.  :/

Reply:
You can have the space, but still make it one string literal

```
" <i class=\"...\">"
```

Although I wonder if it shouldn't be an `&nbsp;` or `&thinsp;`?

Reply:
> You can have the space, but still make it one string literal

That works.  I had tried that before and it didn't work for some reason.  Probably error on my part.  :)